caffeine/src/main/java/com/percolate/caffeine/DialogUtils.java

```diff
@@ -28,15 +31,16 @@
      * @return AlertDialog that is being displayed.
      */
     public static AlertDialog quickDialog(final Activity context, final String message) {
+        if(dialog != null && dialog.isShowing()) return null;
```

Comment:
If we're just keeping a single dialog up-to-date with the latest message sent to this function, then returning `dialog` would make sense.  Returning `null` is going to cause NPEs for most people, including our code.